sdk_container/src/third_party/coreos-overlay/app-emulation/wa-linux-agent/files/0001-flatcar-changes.patch

```diff
@@ -419,6 +421,7 @@ index 00000000..d0d6f7c8
 +# /usr/share/flatcar/etc.
 +#
 +ExecStartPre=/bin/bash -c 'if [[ ! -e /etc/waagent.conf ]]; then ln -sf ../usr/share/waagent/waagent.conf /etc/waagent.conf; fi'
++ExecStartPre=/bin/bash -c 'if [[ ! -e /usr/share/oem/waagent.conf ]]; then ln -sf ../../../etc/waagent.conf /usr/share/oem/waagent.conf; fi'
```

Comment:
That relative link is there on purpose because it also happens to work for `/oem/waagent.conf` or would an absolute link be better?

Reply:
oh you're right, i hadn't thought of it. it's probably better to make this an absolute link

Reply:
Do we need to care about `/usr/share/oem`? I think that just `/oem` should work, since `/usr/share/oem` is a symlink to `/oem`.

Reply:
switched to an absolute symlink in /oem/waagent.conf